<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BossMode</class>
 <widget class="QWidget" name="BossMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>621</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {background-color:rgb(255,132,139,0); broder-radius:3px;color:rbg(0,0,0);}|</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_BossHealth">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>600</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_PlayerHealth">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>630</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_PlayerStamina">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>660</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="AttackButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Attack</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>630</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>621</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="test1.qrc">:/Boss/Image/Balana.jpg</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="test1.qrc"/>
 </resources>
 <connections/>
</ui>
